src/containers/permissionssummary/PermissionsSummaryReducer.js

```diff
@@ -10,7 +8,7 @@ export const INITIAL_STATE:Immutable.Map<*, *> = Immutable.fromJS({
   allRolesList: [],
   entityUserPermissions: [],
   entityRolePermissions: {},
-  propertyPermissions: {},
+  propertyPermissions: Immutable.Map(),
```

Comment:
this shouldn't make a difference. `fromJS` should convert `{}` into `Immutable.Map`

Reply:
True, switching it back.